io-sim/src/Control/Monad/IOSim/CommonTypes.hs

```diff
@@ -28,6 +29,7 @@ module Control.Monad.IOSim.CommonTypes
   , BlockedReason (..)
   , Labelled (..)
   , ppLabelled
+  , IOSimUnique(..)
```

Comment:
```suggestion
  , IOSimUnique
```

Reply:
`base` doesn't export the constructor, can we hide it too?

Reply:
It's properly hidden when re-exporting from `Control.Monad.IOSim`; I only expose the constructor internally to bump uniques in each `schedule`.

Reply:
ok. small style change:
```suggestion
  , IOSimUnique (..)
```

Reply:
Done.